Header header
string group
geometry_msgs/Pose pose
float64[] positions
duration execution_time
